kik_unofficial/utilities/blockhash.py

```diff
@@ -1,41 +1,49 @@
+#! /usr/bin/env python
+
 #
 # This file is taken from https://github.com/commonsmachinery/blockhash-python/blob/master/blockhash.py
 #
 
-#! /usr/bin/env python
 #
 # Perceptual image hash calculation tool based on algorithm descibed in
 # Block Mean Value Based Image Perceptual Hashing by Bian Yang, Fan Gu and Xiamu Niu
 #
 # Copyright 2014 Commons Machinery http://commonsmachinery.se/
 # Distributed under an MIT license, please see LICENSE in the top dir.
 
+# Slight modifications by @feba66 to make black and flake8 happy.
```

Comment:
OK we can leave this comment if you wish

Reply:
Just wanted to distinguish between original code and the modified version, i can remove my name, i dont care about that.

Reply:
Oh OK Now I understood why you added it